apps/src/lib/ui/SchoolInfoConfirmationDialog.jsx

```diff
@@ -78,21 +78,14 @@ class SchoolInfoConfirmationDialog extends Component {
 
   handleClickSave = async () => {
     const {authTokenName, authTokenValue} = this.props.scriptData;
-    fetch(
-      `/api/v1/user_school_infos/${
-        this.props.scriptData.existingSchoolInfo.id
-      }/update_end_date`,
-      {method: 'PATCH', headers: {[authTokenName]: authTokenValue}}
-    )
-      .then(() => {
-        fetch(
-          `/api/v1/users/${
-            this.props.scriptData.existingSchoolInfo.id
-          }/update_school_info_id`,
-          {method: 'PATCH', headers: {[authTokenName]: authTokenValue}}
-        ).then(() => this.props.onClose());
-      })
-      .catch(() => {});
+    fetch(`/api/v1/user_school_infos`, {
```

Comment:
Should there be an id in this route?

Reply:
Intentionally omitted - for now, we're only providing support for a user to update their own `UserSchoolInfo` via this route, and it may or may not create a new entry, so we just post appropriate parameters to this route and let it decide what to do based on context.